content/knitr/options.md

```diff
@@ -6,79 +6,79 @@ slug: options
 show_toc: true
 ---
 
-The **knitr** package shares most options with Sweave, but some were
-dropped or changed, and some new options were added. The default values are in
-the parentheses below. Note that the chunk label for each chunk is assumed
-to be unique, i.e., no two chunks should share the same label. This is especially
+The **knitr** package has most of the same options as Sweave, but some were
+dropped or changed and some new options were added. The default values are in
+parentheses below. Note that the chunk label for each chunk is assumed
+to be unique, i.e. no two chunks should share the same label. This is especially
 important for cache and plot filenames. Chunks without labels will be
 assigned labels like `unnamed-chunk-i` where `i` is an incremental number.
 
 # Chunk Options
 
-Take Rnw files as an example: usually we write chunk options in the form
+Take Rnw files as an example, usually we write chunk options in the form
 `tag=value` like this:
 
 ````r 
 ```{r, mychunk, cache=TRUE, eval=FALSE, dpi=100}
 ```
 ````
 
-And `opts_chunk$set()` can change the default global options in a document
-(e.g. put this in a code chunk: `knitr::opts_chunk$set(comment=NA, fig.width=6,
+And `opts_chunk$set()` can change the default global options of a document
+(e.g., put this in a code chunk: `knitr::opts_chunk$set(comment=NA, fig.width=6,
 fig.height=6)`). A few special notes on the options:
 
-1. Chunk options must be written in one line; no line breaks are allowed
-  inside chunk options;
-1. Avoid spaces ` ` and periods `.` in chunk labels and directory names; if
-  your output is a TeX document, these characters can cause troubles (in
+1. Chunk options must be written in one line and no line breaks are allowed
+  inside chunk options.
+1. Avoid spaces ` ` and periods `.` in chunk labels and directory names. If
+  your output is a TeX document, these characters can cause problems (in
   general it is recommended to use alphabetic characters with words
-  separated by `-` and avoid other characters), e.g. `setup-options`
+  separated by `-` and avoid other characters). For example, `setup-options`
   is a good label, whereas `setup.options` and `chunk 1` are bad;
-  `fig.path='figures/mcmc-'` is a good prefix for figure output, and `fig.path='markov chain/monte carlo'` is bad;
-  non-alphanumeric characters except `-` and `_` in figure filenames will be
-  replaced with `_` automatically;
+  `fig.path='figures/mcmc-'` is a good prefix for figure output and `fig.path='markov chain/monte carlo'` is bad.
+  Non-alphanumeric characters except `-` and `_` in figure filenames will be
+  replaced with `_` automatically.
```

Comment:
Actually this is true only for Rnw documents, so I'd rather not mention the substitution at all.